docs/style-guide.rst

```diff
@@ -874,6 +874,27 @@ No::
     x = y+z;
     x +=1;
 
+***************
+Order of Layout
+***************
+
+We recommend you layout contract elements in the following order:
```

Comment:
The "we-form" is not used anywhere else on this page, I think it mostly uses imperative.